packages/stats/gbstats/bayesian/tests.py

```diff
@@ -57,6 +62,12 @@ class GaussianBayesianConfig(BayesianConfig):
     epsilon: float = 1e-4
 
 
+@dataclass
+class EffectBayesianConfig(BayesianConfig):
+    prior_effect: GaussianPrior = field(default_factory=GaussianPrior)
+    epsilon: float = 1e-4
```

Comment:
Epsilon is used in the lognormal approximation to deal with possibly negative control means, but I don't think we need or use it in the Effect test class, right? Why do we need it here?

Reply:
removed it